app/cloud.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package app
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+func (a *App) CheckAndSendUserLimitWarningEmails() *model.AppError {
+	if a.Srv().License() != nil && !*a.Srv().License().Features.Cloud {
+		// Not cloud instance, do nothing
+		return nil
+	}
+
+	// subscription, subErr := a.Cloud().GetSubscription()
+	// if subErr != nil {
+	// 	return subErr
+	// }
+
+	// if subscription != nil && subscription.IsPaidTier == "true" {
+	// 	// Paid subscription, do nothing
+	// 	return nil
+	// }
```

Comment:
This is commented out so we can test and get design review in a spinwick. Will uncomment before merging.